### PR TITLE
Pointgrey camera

### PR DESCRIPTION
**Pull request by barulicm**
Adds launch files and calibration for the new BlackFly camera.

### CODE REVIEW

File: rr_platform/launch/camera_calibration_18064461.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  data: [730.089675, 0.000000, 662.572253, 0.000000, 734.015999, 486.144443, 0.000000, 0.000000, 1.000000]
+distortion_model: plumb_bob
+distortion_coefficients:
+  rows: 1
+  cols: 5
+  data: [-0.247334, 0.049363, 0.000464, 0.000510, 0.000000]
+rectification_matrix:
+  rows: 3
+  cols: 3
+  data: [1.000000, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000, 0.000000, 0.000000, 1.000000]
```
Comment (ebretl) [inline rr_platform/launch/camera_calibration_18064461.yaml]: This is fine, I don't think we need to worry about rectification for this summer

Comment (barulicm) [reply rr_platform/launch/camera_calibration_18064461.yaml]: The rectification matrix just has to do with stereo camera pairs.
The distortion coefficients, camera matrix, and projection matrix work together to rectify monocular systems.

Comment (ebretl) [reply rr_platform/launch/camera_calibration_18064461.yaml]: So it's actually doing proper rectification?

Comment (barulicm) [reply rr_platform/launch/camera_calibration_18064461.yaml]: Yeah
`/pointgrey_camera/image_color_rect` will give you an undistorted image (the whole, real straight lines look straight bit).

Comment (ebretl) [reply rr_platform/launch/camera_calibration_18064461.yaml]: Nice

**Pull request merged**